### PR TITLE
Fix bug 1226642: Migrate Prism to Bower

### PR DESCRIPTION
**Pull request by openjck**
Before this change, we used the Prism website to create a custom build
of the library. The website generated two minified files, prism.css and
prism.js, which we checked into the repository.

This change uses Bower to cherry-pick the individual components of the
library that we need. Pipeline then builds the components into complete
bundles like the ones we had before. The result is the same, but this
approach helps us to be more clear about what version of Prism we use
and what we include in our custom build.

As with other libraries managed by Bower, the library code is checked
into the repository but the minified bundles are not.

### CODE REVIEW

File: kuma/static/styles/libs/prism/prism-line-numbers.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	counter-reset: linenumber;
+}
+
+pre.line-numbers > code {
+	position: relative;
+}
+
+.line-numbers .line-numbers-rows {
+	position: absolute;
+	pointer-events: none;
```
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-numbers.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/styles/libs/prism/prism-line-numbers.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+
+}
+
+	.line-numbers-rows > span {
+		pointer-events: none;
```
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-numbers.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/styles/libs/prism/prism-line-highlight.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	padding: inherit 0;
+	margin-top: 1em; /* Same as .prism’s padding-top */
+
+	background: hsla(24, 20%, 50%,.08);
+	background: -moz-linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	background: -webkit-linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	background: -o-linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	background: linear-gradient(left, hsla(24, 20%, 50%,.1) 70%, hsla(24, 20%, 50%,0));
+	
+	pointer-events: none;
```
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-highlight.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-highlight.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

**Comment by stephaniehobson** [comment]
Looking good in both production and dev mode. 

Several unrelated rabbit holes found.

R+

File: kuma/static/styles/libs/prism/prism-line-highlight.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-highlight.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/styles/libs/prism/prism-line-numbers.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-numbers.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/styles/libs/prism/prism-line-numbers.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-numbers.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/styles/libs/prism/prism-line-numbers.css (hunk 1 above)
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-numbers.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/styles/libs/prism/prism-line-numbers.css (hunk 2 above)
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-numbers.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

File: kuma/static/styles/libs/prism/prism-line-highlight.css (hunk 3 above)
Comment (mdn-services) [inline kuma/static/styles/libs/prism/prism-line-highlight.css]: CSS pointer-events (for HTML) not supported by: IE (10), IE Mobile (10)

**Pull request merged**